sdk/lib/atl/cstringt.h

```diff
@@ -533,6 +533,43 @@ class CStringT :
         return str2.GetLength() == 1 && str2[0] == ch1;
     }
 
+    friend bool operator!=(const CStringT& str1, const CStringT& str2) throw()
```

Comment:
Hmm... Why not just calling the corresponding `operator==` and NOT-ing the result?

Reply:
because this was easier.

Reply:
Copy-pasting the code from operator== and manually changing every condition to its negative was "easier"??

Reply:
yes.